fast_response/MonitoringAndMocks/Data_Display.py

```diff
@@ -8,756 +8,387 @@
-ax.plot_date(unique_days, update, color = 'red')
-
-ax.fmt_xdata = DateFormatter('%Y-%m-%d')
-plt.xticks(rotation=18, ha="right")
-
-plt.legend(["Total", "Preliminary", "Initial", "Update"], loc = "upper left")
-
-plt.title("Reports per Day")
-plt.xlabel("Date")
-plt.ylabel("Number of Reports")
-plt.ylim([0,70])
-
-save_path='/home/mromfoe/public_html/O4_followup_monitoring/ReportsPerDay_liveupdate.png'
-
-plt.tight_layout()
-fig.savefig(save_path)
-fig.savefig("ReportsPerDay.png")
-
-###Find average time between mocks/runs
-
-time_average = []
-for i in range(len(First_Batch["Trigger_Time"]))[:-1]:
-        delta_t = First_Batch["Trigger_Time"][i+1] - First_Batch["Trigger_Time"][i]
-        time_average.append(delta_t*86400)
-
-fig, axs = plt.subplots(figsize = (10, 7))
-
-plt.xlabel("Time Between Runs in Seconds")
-plt.ylabel("# of Runs")
-plt.title("Average Delta T")
-
-time_average = np.array(time_average)
-time_average[time_average > 8000] =  8000
-h,b,_= axs.hist(time_average, bins= np.linspace(0, 8000, 25))
-med_delta = median(time_average)
-
-tallest = max(h)
-
-plt.axvline(med_delta, color = 'k', linestyle = 'dashed', label = "Median Delta T")
-plt.text(med_delta*1.3, tallest*0.8, 'Median: {:.2f}'.format(med_delta), fontsize = "15")
-
-plt.legend(fontsize = "18")
-
-axs.grid(b = True, color ='black',
-        linestyle ='-.', linewidth = 0.5,
-        alpha = 0.7)
-
-plt.tight_layout()
-fig.savefig("Linear_Delta_T.png")
-
-time_average_too = []
-for i in range(len(First_Batch["Time_Stamp"]))[:-1]:
-        delta_t = Time(First_Batch["Time_Stamp"][i+1]).mjd - Time(First_Batch["Time_Stamp"][i]).mjd
-        time_average_too.append(delta_t*86400)
-
-fig, axs = plt.subplots(figsize = (10, 7))
-
-plt.xlabel("Time Between Runs in Seconds")
-plt.ylabel("# of Runs")
-plt.title("Average Delta T")
-
-time_average_too = np.array(time_average_too)
-time_average_too[time_average_too > 8000] =  8000
-h,b,_= axs.hist(time_average_too, bins= np.linspace(0, 8000, 25))
-med_delta = median(time_average_too);
+save_path='/home/mromfoe/public_html/O4_followup_monitoring/Update_Time.png'
+img.save(save_path)
+img.save("Update_Time.png")
+
+df = pd.DataFrame({"Name": ed["Name"][:-16:-1],
+                    "Merger Time": ed["Trigger_Time"][:-16:-1],
+                    "GCN Alert": ed["GCN_Alert"][:-16:-1],
+                    "Script Finishes": ed["End_Time"][:-16:-1],
+                    "Total Latency in Seconds": ed["Total_Latency"][:-16:-1]})
+html1 = df.to_html()
 
-tallest = max(h)
+text_file1 = open("/home/mromfoe/public_html/O4_followup_monitoring/Recent_Runs.html", "w")
+text_file1.write(html1)
+text_file1.close()
 
-plt.axvline(med_delta, color = 'k', linestyle = 'dashed', label = "Median Delta T")
-plt.text(med_delta*1.3, tallest*0.8, 'Median: {:.2f}'.format(med_delta), fontsize = "15")
+for event in range(len(ed)):
+    if ed["first"][event] == True:
+        df = pd.DataFrame({"Name": ed["Name"][event],
```

Comment:
mine is throwing an error here, because these are each scalars. there should be a way to select the first ones and make a little dataframe the way you do above, but for first events only, and then do the `df.to_html()` call. Let me know if you need help doing this here